engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -92,6 +95,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   }
 
   // When PreparingJob, the FSM always has NoData
+  //RUCHI: Note, the backend preparation actor failed to intialize a few times, make a ticket or figure out whats up
```

Comment:
I made this a bit more resilient in my EJEASpecs PR... not sure whether that'll actually help you though!